DB-scripts/CreateBase.sql

```diff
@@ -96,7 +96,9 @@ INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'like', 'li
 commit;
 INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'location', 'location');
 commit;
-INSERT INTO each_prop (eid, name, type) VALUES (NEXTVAL('each_seq'), 'museum', 'museum');
```

Comment:
Why did you delete museum property?

Reply:
Cause it didn't exsist at the screenshot I sent to you. I can add them if you want

Reply:
You have the correct file